src/util.iced

```diff
@@ -1,7 +1,11 @@
+stream = require('stream')
```

Comment:
I doubt it, that'd require changing a bunch of callers. Besides, consumers of this library will probably never need to touch this file